### ISSUE TITLE
Custom install option cannot procceed

### ISSUE THREAD

**Issue by KomradBear** [elementary/installer#618]
## What Happened
When trying to do a custom install, no matter what kind of partitioning I do, the "Erase & Install" button stays inactive.
The default I always do on my SSD is 50mb of fat32 for efi, and the rest as btrfs for OS.
I then select the partitions and check the "use partition" both for boot as efi on the fat32 partition and as root / for the btrfs

## Platform Information
elementary OS 6 odin.

**Comment by vjr** [comment]
Does the installer not inform you that you need an ESP (EFI partition) at least 512 MiB ?

**Comment by KomradBear** [comment]
No, I just tried it and it seems like 260mib is the minimum that works.
The installer doesn't inform about this restriction in any way... that was extremely frustrating to say the least.

**Comment by nurullahsaeem** [comment]
I'm having the same problem. Did you manage to fix it?

**Comment by iouae** [comment]
Any fixes? Still no luck.

**Comment by lheair** [comment]
On one HD I have 2 versions of Windows, 2 mac OS, 4 Linux distros. All multibooted via rEFInd. All install smoothly and work flawlessly. Comes elementary OS, which, for some unclear reason decided to change its installation process to a different one, one that demands an EFI partition that is at least 256MB.
Needless to say, I don't particularly want to re-partition my drive and re-install all the OSes because elementary OS cannot be satisfied with a 200 MB EFI partition, like EVERY other OS I use. And why does it need a bigger EFI partition? No apparent reason!
Almost a year later and elementary OS has obviously no plans to fix this so it's goodbye elementary OS.

**Comment by vjr** [comment]
Have you tried just adding a second EFI (ESP) partition and installing elementary OS? EFI spec (I understand) allows multiple ESPs and rEFInd might support it?

**Comment by lheair** [comment]
The thing is, my HD is already partitioned with every partition used. There is no room for a second EFI partition. I could resize the partition that follows the EFI one but in my own experience, Gparted doesn't understand hybrid partitioning (MBR and GPT) and will mess things up to the point I might have to start all over re-installing everything.

Considering how many people have had problems with this requirement from elementary OS, for which I see no reason, I was hoping it would have been fixed by now, by simply removing the minimum 256 MB restriction. There is no way elementary OS needs that much space on the EFI partition.
